Dockerfile

```
@@ -0,0 +1,9 @@
+FROM openjdk:17-slim
+
+WORKDIR /app
+ARG JAR_FILE=build/libs/warehouse-0.0.1-SNAPSHOT.jar
```

```suggestion
ARG JAR_FILE=build/libs/warehouse-*.jar
```

don't hardcode version